wd_demo.au3

```diff
@@ -1263,3 +1264,50 @@ Func SetupOpera($bHeadless)
 	Local $sCapabilities = _WD_CapabilitiesGet()
 	Return $sCapabilities
 EndFunc   ;==>SetupOpera
+
+Func SetupEdgeIEMode() ; this is for MS Edge IE Mode
+	#REMARK In order to properly use MS Edge In IE Mode you need To Do some settings - Set the "Internet, Local intranet, Trusted sites, Restricted sites" to protected mode, enabling or disabling it is not a matter
+	; https://www.edureka.co/community/1690/problem-using-internetexplorerdriver-selenium-webdriver
+	Local $sTimeStamp = @YEAR & '-' & @MON & '-' & @MDAY & '_' & @HOUR & @MIN & @SEC
+	; https://www.selenium.dev/documentation/ie_driver_server/#required-configuration
+	_WD_Option('Driver', 'IEDriverServer.exe') ;
+	_WD_Option('Port', 5555)
+	_WD_Option('DriverParams', '--verbose --log-path="' & @ScriptDir & '\' & $sTimeStamp & ' EdgeIEMode.log" -log-level=INFO' & " --port=" & $_WD_PORT & " -host=127.0.0.1" )
```

Comment:
- According to [here](https://www.selenium.dev/documentation/ie_driver_server/), this should be `-log-file` instead of `--log-path`. 
- It doesn't look like `--verbose` is valid either.
- Need to check `-port` vs `--port`.

Reply:
> * According to [here](https://www.selenium.dev/documentation/ie_driver_server/), this should be `-log-file` instead of `--log-path`.
> * It doesn't look like `--verbose` is valid either.

already tested:
any change you pointed me to, will effect in not creating log files.

Reply:
Hmmm... suggest that you check again because I just tried it with the recommended switch and the log file was successfully created.

Reply:
Check this commit: [DriverParams usage](https://github.com/Danp2/au3WebDriver/pull/454/commits/36486fa731192afe31f6fe7f0d66549c521b5462)